requirements.txt

```diff
@@ -1,2 +1,2 @@
-pandas
+numpy>=1.19.0
```

Comment:
@IAlibay is bumping the version to 1.20.0 for MDAnalysis in https://github.com/MDAnalysis/mdanalysis/pull/3737

Reply:
Doesn't impact things here so it wouldn't think it matters. If you want to raise numpy to 1.20 you'll have to drop py3.6.

Reply:
What should we prioritise here? Staying in-step with MDAnalysis' dependencies or keeping Py3.6?

Reply:
It doesn't impact MDAnalysis if panedr supports a wider range of python & numpy versions, so IMHO it's fine to just leave it as-is.